web/src/app/services/ServiceForm.tsx

```diff
@@ -1,13 +1,32 @@
 import React from 'react'
-import p from 'prop-types'
 import Grid from '@material-ui/core/Grid'
 import TextField from '@material-ui/core/TextField'
 import { EscalationPolicySelect } from '../selection/EscalationPolicySelect'
 import { FormContainer, FormField } from '../forms'
 
-export default function ServiceForm(props) {
-  const { epRequired, ...containerProps } = props
+interface Value {
+  name: string
+  description: string
+  escalationPolicyID: string
```

Comment:
while `Value` should be required, these three keys inside of `Value` should be optional

Reply:
Made requested changes and re-commited.